### DISCUSSION TITLE
[Emergency]Error in extracting exposure instruments, telling to contact the developers.

### DISCUSSION THREAD

**Discussion by TJJTJ** [MRCIEU/TwoSampleMR#501] [Q&A] [answered]
>vc_exp <- extract_instruments(outcomes = "ukb-b-12466")
Error in tophits_query(id = id, pval = pval, clump = clump, r2 = r2, kb = kb,  : 
  There was an error, please contact the developers

NEED FOR YOUR HELP!!!THX!!!!

**Comment by N3therwind** [comment]
I've encountered the same issue as you. May I ask if you have resolved it?

**Reply by TJJTJ** [reply]
Not yet... Such an annoying problem. Have you found other solutions?

**Reply by N3therwind** [reply]
I used read_exposure_data() to read the data from a local source, perhaps you could give that a try as well...

**Reply by TJJTJ** [reply]
Okay thanks, I'll try it later, maybe it's the only useful idea right now.

**Reply by explodecomputer** [reply, marked as answer]
Hey all have a go at installing the latest GitHub version of ieugwasr. I'll try to release it to cran soon.

The issue arises when the GWAS has no instruments.

We've been making a lot of changes to improve server stability so that has introduced some other problems like this one, apologies.
